jscomp/build_tests/react_ppx/src/recursive_component_test.bs.js

```diff
@@ -11,8 +11,11 @@ function make(Props) {
             });
 }
 
-function make$1(props) {
-  return Curry._1(make$1, {});
+function make$1(_props) {
+  while(true) {
+    _props = {};
+    continue ;
+  };
```

Comment:
Not sure why this js output diff has made after running `make test`

Reply:
It should not happen. Try make clean and make test again.